bindings/go/src/fdb/futures.go

```diff
@@ -366,27 +407,40 @@ type FutureStringSlice interface {
 
 type futureStringSlice struct {
 	*future
+	o sync.Once
+	e error
+	v []string
```

Comment:
I would suggest using `once` instead of `o` (`once.Do` reads much better) and `err` instead of `e` in field names (I see `err` used at other parts of this file). I dunno what `v` is which probably is good indicator that it should be renamed too.

Reply:
In that case, it is the value the future will resolve with.

I agree with your assessment about the names, but I don’t know if this PR is the place to resolve it.

Reply:
Ah ok. I missed that it was that way already in other part of codebase.